app/database/schema.py

```diff
@@ -28,29 +29,18 @@ def validate(cls, v):
     def __modify_schema__(cls, field_schema):
         field_schema.update(type='string')
 
-
 #creating channel class to store schema 
 class Message(BaseModel):
-    """
-     This Class is the Schema for our data model 
-    
-     Schema :
-        \n_id -> Object ID,
-        \nName -> String,
-        \nMessage -> String,
-    
-     Updated Schema for next week:
-    
-        0. Object_id
-        1. sender_id
-        2. channel_id
-        3.message_body
-        4.time_stamp
-    
+    """[summary]
```

Comment:
there's no point in giving docstrings if you are going to leave it like this. Better habit would be to write doc strings first then proceed with writing the code. It gives you much better clarity of your logic of what you will do.